doc/tutorials/content/compiling_pcl_posix.rst

```diff
@@ -86,17 +86,18 @@ The following code libraries are **required** for the compilation and usage of t
 .. note::
 pcl_* denotes all PCL libraries, meaning that the particular dependency is a strict requirement for the usage of anything in PCL.
 
-+---------------------------------------------------------------+-----------------+-------------------+-------------------+
-| Logo                                                          | Library         | Minimum version   | Mandatory         |
-+===============================================================+=================+===================+===================+
-| .. image:: images/posix_building_pcl/boost_logo.png           | Boost           | 1.54              | pcl_*             |
-+---------------------------------------------------------------+-----------------+-------------------+-------------------+
-| .. image:: images/posix_building_pcl/eigen_logo.png           | Eigen           | 3.0               | pcl_*             |
-+---------------------------------------------------------------+-----------------+-------------------+-------------------+
-| .. image:: images/posix_building_pcl/flann_logo.png           | FLANN           | 1.7.1             | pcl_*             |
-+---------------------------------------------------------------+-----------------+-------------------+-------------------+
-| .. image:: images/posix_building_pcl/vtk_logo.png             | VTK             | 5.6               | pcl_visualization |
-+---------------------------------------------------------------+-----------------+-------------------+-------------------+
++---------------------------------------------------------------+-----------------+-------------------------+-------------------+
+| Logo                                                          | Library         | Minimum version         | Mandatory         |
++===============================================================+=================+=========================+===================+
+| .. image:: images/posix_building_pcl/boost_logo.png           | Boost           | | 1.40 (without OpenNI) | pcl_*             |
+|                                                               |                 | | 1.47 (with OpenNI)    |                   |
++---------------------------------------------------------------+-----------------+-------------------------+-------------------+
```

Comment:
It's ugly but I did not find a better way to break lines:
![capture du 2015-02-08 14 58 56](https://cloud.githubusercontent.com/assets/5566160/6096757/25e7de62-afa3-11e4-9e7a-12aabc629119.png)